src/Native/WebGL.js

```diff
@@ -51,44 +51,52 @@ var _elm_community$webgl$Native_WebGL = function () {
 
   }
 
+ /**
+  *  Apply settings to the gl context
+  *
+  *  @param {WebGLRenderingContext} gl context
+  *  @param {List} settings the list of settings coming in from Elm
+  *  @return {Array<GLenum>} the list of enabled capabilities, caused by
+  *          applying the settings
+  */
   function doSettings(gl, settings) {
-    var disable = [];
+    var enabled = [];
```

Comment:
I think I'd just call it `enabledCapabilities` - if I was looking at this function in isolation, that name would be obvious to me what it does